web/src/search/input/toggles/Toggles.tsx

```diff
@@ -89,8 +91,18 @@ export const Toggles: React.FunctionComponent<TogglesProps> = (props: TogglesPro
         submitOnToggle({ newPatternType })
     }
 
+    const fullQuery = [
+        props.navbarSearchQuery,
+        props.filtersInQuery && generateFiltersQuery(props.filtersInQuery),
+        `patternType:${props.patternType}`,
+        props.caseSensitive ? 'case:yes' : '',
+    ]
+        .filter(query => query && query.length > 0)
```

Comment:
If `query` is a string that would be sufficient
```suggestion
        .filter(query => !!query)
```